### PR TITLE
update webb-proposals tests and add pallet-signature-bridge to runtime #209

### PR DESCRIPTION
**Pull request by salman01zp**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Task CheckList
Update Test cases for Pallets to use webb-proposals
   - [x]  Update Test case for `TokenWrapperHandler`
   - [x]  Update Test case for `AnchorHandler`
   - [x]  fix `pallet-xanchor` to support webb-proposals.
   - [x]   Update Test case for `XAnchor`
   - [x]  Add `SignatureBridge` Pallet to runtime

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [x] Yes
- [ ] No

- update xanchor pallet with new webb-proposal changes

**Reference issue to close (if applicable)**

Closes #211 #213

### CODE REVIEW

File: pallets/anchor-handler/src/tests_signature_bridge.rs [hunk 1]
```diff
@@ -85,6 +85,15 @@ fn get_public_uncompressed_key() -> [u8; 64] {
 	hex!("8db55b05db86c0b1786ca49f095d76344c9e6056b2f02701a7e7f3c20aabfd913ebbe148dd17c56551a52952371071a6c604b3f3abe8f2c8fa742158ea6dd7d4")
 }
 
+fn make_proposal_header(
+	resource_id: webb_proposals::ResourceId,
+	nonce: webb_proposals::Nonce,
+) -> webb_proposals::ProposalHeader {
+	let function_signature = webb_proposals::FunctionSignature::new(hex_literal::hex!("cafebabe"));
```
Comment (shekohex) [inline pallets/anchor-handler/src/tests_signature_bridge.rs:92]:
```suggestion
	let function_signature = webb_proposals::FunctionSignature::new([0u8; 4]);
```

**Comment by salman01zp** [comment]
Will fix webb_proposal dependency once new version(v0.3.1) is published

**Comment by salman01zp** [comment]
Will fix webb_proposal dependency once new version(v0.3.1) is published

**Review by shekohex — CHANGES_REQUESTED** [review]

**Comment by shekohex** [comment]
> Will fix webb_proposal dependency once new version(v0.3.1) is published

I will publish a new version now.

**Review by shekohex — APPROVED** [review]
LGTM!

**Pull request merged**